### DISCUSSION TITLE
Eclipse Theia v1.53.0

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia#14088] [Release Announcements] [answered]
Hi everyone :wave:,

The Eclipse Theia v1.53.0 release is scheduled for later today!
We will inform you once the release starts and completes.

**Comment by msujew** [comment]
@jfaltermeier Please take a look at https://github.com/eclipse-theia/theia/pull/14092 before performing the release.

**Comment by jfaltermeier** [comment]
The release will start now, I’ll post an update when it has completed.

**Comment by jfaltermeier** [comment]
The release has completed, thank you to everyone who participated and contributed!

**Comment by JonasHelming** [comment, marked as answer]
[News & Noteworthy](https://eclipsesource.com/blogs/2024/09/09/eclipse-theia-1-53-release-news-and-noteworthy/)
